integration-tests/src/main/features/opentelemetry-java.json

```diff
@@ -0,0 +1,93 @@
+{
+  "id": "${project.groupId}:${project.artifactId}:slingfeature:opentelemetry-java:${project.version}",
+  "bundles": [
+    {
+      "id": "${project.groupId}:opentelemetry-api:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-api-incubator:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-context:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-common:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-logs:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-metrics:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-trace:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-extension-autoconfigure:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-extension-autoconfigure-spi:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-sdk-extension-incubator:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-exporter-common:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-exporter-logging:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-exporter-otlp:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "${project.groupId}:opentelemetry-exporter-otlp-common:${opentelemetry.java.version}.000-SNAPSHOT",
+      "start-order": 5
+    },
+    {
+      "id": "com.fasterxml.jackson.core:jackson-core:2.17.2",
+      "start-order": 5
```

Comment:
should thse all have start order 5?

Reply:
they are all required for opentelemetry to work, so i think that it's indeed best to set it to 5